src/nft-card.ts

```diff
@@ -104,30 +121,37 @@ export class NftCard extends LitElement {
         await this.requestUpdate()
 
         // Watch for the account to change then update state of component
-        window.ethereum.on('accountsChanged', function (accounts) {
-            [this.account] = accounts
-            console.log('accountsChanged', accounts)
-        })
+        // window.ethereum.on('accountsChanged',  (accounts: Array<string>) => {
```

Comment:
why did this get removed?

Reply:
I am putting this on the backburner as the component is not reacting to the changing account. The plan is to spend some time investigating why this is, as well as, consulting the Polymer team in slack. Once component is in v1 then I want to address this. Sound good?